README.md

```diff
@@ -193,14 +193,150 @@ developer-portal 📂
 │       │    │    │    img1.png
 │       │    │    │    img2.png
 │       │    │    │    ...   
+│       │    │  
+│       │    └───api 📂
+│       │    │    │    
+│       │    │    │    ...  
+│   
+│   
+│   
+└───Product 2 📂
+    │  
+    │  ...
+```
+
+### API folder
+
+Finally, for each api folder you have do the following:
+
+* Create `docs` folder to store `*.md` documentation of your API
+* Create `images` folder to store api-level images
+* Create [`api.yaml`](#apiyaml-file) file.
+
+**Final folder structure** should look like this:
+
+```
+developer-portal 📂
+│
+└───Product 1 📂
+│   │   concepts.md
+│   │   terminology.md
+│   │   index.md
+│   │   product.yaml
+│   │  
+│   └───docs 📂
+│   │   │   doc1.md
+│   │   │   doc2.md
+│   │   │   ... 
+│   │   
+│   └───images 📂
+│   │   │   img1.png
+│   │   │   img2.png
+│   │   │   ... 
+│   │  
+│   └───services 📂
+│       │  
+│       └───Service 1 📂
+│       │    │  
+│       │    │   index.md
+│       │    │   
+│       │    └───docs 📂
+│       │    │    │    doc1.md
+│       │    │    │    doc2.md
+│       │    │    │    ...     
+│       │    │    
+│       │    └───images 📂
+│       │    │    │    img1.png
+│       │    │    │    img2.png
+│       │    │    │    ...   
+│       │    │  
+│       │    └───api 📂
+│       │    │    │    
+│       │    │    │  api.yaml
+│       │    │    │     
+│       │    │    └───docs 📂
+│       │    │    │    doc1.md
+│       │    │    │    doc2.md
+│       │    │    │    ...     
+│       │    │    
+│       │    └───images 📂
+│       │    │    │    img1.png
+│       │    │    │    img2.png
+│       │    │    │    ...   
+│       │    │    
 │   
 │   
+└───Product 2 📂
+    │  
+    │  ...
+```
+
+### Private docs
+
+Developer portal allows you creating private docs. These docs will only be visible to authenticated Extenda users.
+
+Use private docs to explain architecture, 
```

Comment:
This is not a full sentence :smile:
Are you missing the information, or do you not have the information needed to write the paragraph?

Reply:
Fixed it 🤣